core/shared/platform/include/platform_api_extension.h

```diff
@@ -977,6 +977,36 @@ os_socket_set_broadcast(bh_socket_t socket, bool is_enabled);
 int
 os_socket_get_broadcast(bh_socket_t socket, bool *is_enabled);
 
+/**
+ * Get the peer address of the connected to the socket
+ *
+ * @param socket  [INTPUT] the connected socket
+ * @param addr    [OUTPUT] the address of the peer socket. If addr is NULL,
+ *                nothing is filled in, and addrlen will not be used
```

Comment:
No addrlen argument here? The prototype is different from libc function prototype, and the implementation of esp-idf platform sets the input size as `socklen_t addr_len = sizeof(struct sockaddr_in);`, so could you please add comment about it (like the size of addr should be no smaller than sizeof(struct sockaddr_int)?